netbench/netbench-driver/etc/Dockerfile

```diff
@@ -65,12 +65,18 @@ RUN set -eux; \
   fi; \
   rm -rf target
 
+RUN set -eux; \
+  cd netbench; \
+  cargo run --bin netbench-scenarios; \
+  cp target/netbench/request_response.json /app/bin; \
+  rm -rf target;
+
 ARG DRIVER="s2n-quic"
 ARG ENDPOINT="client"
 
 RUN set -eux; \
   export TARGET="netbench-driver-$DRIVER-$ENDPOINT"; \
-  echo "#!/usr/bin/env bash\neval /usr/bin/netbench-collector /usr/bin/$TARGET \$@" > /app/bin/start; \
+  echo "#!/usr/bin/env bash\nexport SERVER_0=\$(dig +short \$DNS_ADDRESS):\$SERVER_PORT\nprintenv\neval /usr/bin/netbench-collector /usr/bin/$TARGET \$@ > client.json\naws s3 cp ./client.json s3://\$S3_BUCKET/" > /app/bin/start; \
```

Comment:
Ideally the json file would be unique across runs and driver. Something like

```
$RUN_ID/$DRIVER-$ENDPOINT.json
```

This way we can keep a historical record of runs

Reply:
Sounds good, I'll clean up my Dockerfile and re-commit